packages/aws-client/src/SQS.ts

```diff
@@ -139,16 +143,25 @@ export const receiveSQSMessages = async (
   return <SQSMessage[]>(messages.Messages ?? []);
 };
 
+/**
+ * Extract SQS message body
+ *
+ * @param {SQSRecord | AWS.SQS.Message} message - SQS message
+ * @param {string} message.body - optional key for message body
+ * @param {string} message.Body - optional key for message body
+ * @returns {StepFunctionEventBridgeEvent | CumulusMessage} an AWS SQS message body
+ * parsed into a CumulusMessage or EventBridgeEvent
+ */
 export const parseSQSMessageBody = (
   message: SQSRecord | AWS.SQS.Message
-): { [key: string]: any } =>
+): StepFunctionEventBridgeEvent | CumulusMessage | SQSRecord =>
   JSON.parse(get(message, 'Body', get(message, 'body')) ?? '{}');
 
 /**
  * Delete a given SQS message from a given queue.
```

Comment:
This jsdoc is throwing lint on being incomplete: 


```suggestion
/**
 * Deletes a message from an Amazon SQS queue.
 *
 * @param {string} QueueUrl - The URL of the Amazon SQS queue.
 * @param {string} ReceiptHandle - The unique identifier of the SQS message.
 * @returns {Promise} an AWS SQS response.
 * @throws Will throw an error if deletion fails.
 */```

Reply:
I *think* this is corrected now, but no tthe way you said exactly. I'ev corrected the names to be capitalized correctly, but I haven't added the throws line? according to the aws docs this should report errors as http status codes (4xx codes). do you know otherwise?

Reply:
resolved in huddle